sw/airborne/firmwares/rotorcraft/navigation.c

```diff
@@ -110,7 +110,7 @@ static inline void nav_set_altitude(void);
 
 /** minimum horizontal distance to waypoint to mark as arrived */
 #ifndef ARRIVED_AT_WAYPOINT
-#define ARRIVED_AT_WAYPOINT 3.0
+#define ARRIVED_AT_WAYPOINT 0.2
```

Comment:
this should be changed in airframe file since it is way to low for a regular GPS